### PR TITLE
[REF] Refactor `get_suvr_mask`

### PR DESCRIPTION
**Pull request by NicolasGensollen**
Same as #748 but for function `get_suvr_mask`

### CODE REVIEW

**Comment by ghisvail** [comment]
Fixed the conflicts while rebasing on `dev`

**Comment by NicolasGensollen** [comment]
> Fixed the conflicts while rebasing on `dev`

Great! Thanks for taking care of that! 👍

File: test/unittests/utils/test_pet.py [hunk 1]
```diff
@@ -116,3 +116,23 @@ def test_read_psf_information(tmp_path: os.PathLike, psf_df: pd.DataFrame):
...
     ) == [[8, 9, 10], [8, 9, 10]]
+
+
+@pytest.mark.parametrize(
+    "label", ["pons", "cerebellumPons", "pons2", "cerebellumPons2"]
+)
+def test_get_suvr_mask(label: str):
+    from clinica.utils.pet import get_suvr_mask
+
+    assert get_suvr_mask(label).exists()
```
Comment (ghisvail) [inline test/unittests/utils/test_pet.py:127]: "Unresolved attribute reference 'exists' for class 'PathLike'"

```suggestion
    assert Path(get_suvr_mask(label)).exists()
```

File: clinica/utils/pet.py [hunk 2]
```diff
@@ -113,45 +113,34 @@ def read_psf_information(
...
-            os.path.split(os.path.realpath(__file__))[0],
-            "..",
-            "resources",
-            "masks",
-            "region-cerebellumPons_remove-extrabrain_eroded-3it_mask.nii.gz",
-        ),
+    from pathlib import Path
+
+    CURRENT_DIR = Path(os.path.realpath(__file__))
+    MASKS_DIR = CURRENT_DIR.parent.parent / "resources/masks"
```
Comment (ghisvail) [inline clinica/utils/pet.py:133]: "Variable in function should be lowercase"

Comment (ghisvail) [reply clinica/utils/pet.py:133]: also:

```suggestion
    CURRENT_DIR = Path(os.path.realpath(__file__))
    MASKS_DIR = CURRENT_DIR.parent.parent / "resources" / "masks"
```

**Review by ghisvail — CHANGES_REQUESTED** [review]
LGTM, with minor corrections

File: clinica/utils/pet.py [hunk 3]
```diff
@@ -113,45 +113,34 @@ def read_psf_information(
...
-            os.path.split(os.path.realpath(__file__))[0],
-            "..",
-            "resources",
-            "masks",
-            "region-cerebellumPons_remove-extrabrain_eroded-3it_mask.nii.gz",
-        ),
+    from pathlib import Path
+
+    CURRENT_DIR = Path(os.path.realpath(__file__))
+    MASKS_DIR = CURRENT_DIR.parent.parent / "resources" / "masks"
```
Comment (ghisvail) [inline clinica/utils/pet.py:133]:
```suggestion
    current_dir = Path(os.path.realpath(__file__))
    masks_dir = current_dir.parent.parent / "resources" / "masks"
```

File: clinica/utils/pet.py [hunk 4]
```diff
@@ -113,45 +113,34 @@ def read_psf_information(
...
+        "pons2": "region-pons_remove-extrabrain_eroded-2it_mask.nii.gz",
+        "cerebellumPons2": "region-cerebellumPons_remove-extrabrain_eroded-3it_mask.nii.gz",
     }
-    return suvr_reference_region_to_suvr[suvr_reference_region]
+    if suvr_reference_region not in suvr_reference_region_labels_to_filenames:
+        raise ValueError(
+            f"SUVR reference region label {suvr_reference_region} is not supported. "
+            f"Supported values are : {list(suvr_reference_region_labels_to_filenames.keys())}."
+        )
+    return MASKS_DIR / suvr_reference_region_labels_to_filenames[suvr_reference_region]
```
Comment (ghisvail) [inline clinica/utils/pet.py:146]:
```suggestion
    return masks_dir / suvr_reference_region_labels_to_filenames[suvr_reference_region]
```

**Pull request merged**